WordPress/wporg-developer-blog
# Enable and customize open graph tags

This PR force-enables the Jetpack Open Graph behavior, and then runs over the values to customize them slightly. Mostly to make sure they're consistent and accurate for each value/page.

I've used [the main developer.w.org's default image](https://developer.wordpress.org/wp-content/themes/wporg-developer-2023/images/opengraph-image.png), but we can easily swap that out before merging this PR if there's a DevBlog specific one.

Fixes #53 

**Screenshots**

I've been testing by manually checking the generated code, so that's what I'm sharing here.

Homepage: uses the default image, site title, and site description.

<img width="941" alt="home" src="https://github.com/WordPress/wporg-developer-blog/assets/541093/e509d213-04c8-4e5e-8e87-7c2c7e21ef24">

Article, with a featured image: uses the featured image, post title, and excerpt.

<img width="1137" alt="post-with-image" src="https://github.com/WordPress/wporg-developer-blog/assets/541093/750bdd55-f7d8-44bb-87e8-5d8262ce3ee3">

Article without a featured image: uses the default image, post title, and excerpt.

<img width="1136" alt="post-no-image" src="https://github.com/WordPress/wporg-developer-blog/assets/541093/edb1fc97-2266-4bed-a71d-cb927526ba07">

Category (& other archive pages): uses the default image, archive title, and description (omitted if not available or relevant).

<img width="959" alt="category" src="https://github.com/WordPress/wporg-developer-blog/assets/541093/3f0c5f1b-df55-4f5e-9fa4-d56833f27d4c">

**To test**

- Load any page, view source or inspect the head
- Find ``
- The tags should be accurate to the page they are on

(I know there's a way to test locally with a tunnel, but I haven't tried that.)
